File: java-framework/src/main/java/examples/basic/BrowserExample.java
package examples.basic;

import static java.util.concurrent.TimeUnit.SECONDS;
import static webui.automation.framework.BrowserType.FIREFOX;

import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

import webui.automation.framework.Browser;

/**
 * This example tests the {@link Browser} class.
 */
public class BrowserExample 
{
    public static void main( String[] args )
    {
        String url = "http://google.com";

        Browser browser = new Browser();
        browser.open(FIREFOX).navigateTo(url);
        try {
            WebElement searchBox = browser.findElement(By.name("q"));
            searchBox.sendKeys("current movies");
            searchBox.submit();
            browser.sleep(3, SECONDS);
        } finally {
            browser.close();
        }
    }
}

File: cs-examples/BrowserExample/BrowserExample.cs
namespace Examples.Basic
{

//JAVA TO C# CONVERTER TODO TASK: This Java 'import static' statement cannot be converted to C#:
//    import static webui.automation.framework.BrowserType.FIREFOX;

    using By = org.openqa.selenium.By;
    using WebElement = org.openqa.selenium.WebElement;

    using Browser = webui.automation.framework.Browser;

    /// <summary>
    /// This example tests the <seealso cref="Browser"/> class.
    /// </summary>
    public class BrowserExample
    {
        public static void Main(string[] args)
        {
            string url = "http://google.com";

            Browser browser = new Browser();
            browser.Open(FIREFOX).NavigateTo(url);
            try
            {
                WebElement searchBox = browser.FindElement(By.Name("q"));
                searchBox.SendKeys("current movies");
                searchBox.Submit();
                browser.Sleep(3, SECONDS);
            }
            finally
            {
                browser.Close();
            }
        }
    }

}